tests/integration_tests/functional/test_cpu_features.py

```diff
@@ -452,21 +457,13 @@ def _test_cpu_wrmsr_snapshot(context):
     )
 
 
-@pytest.mark.skipif(
-    PLATFORM != "x86_64", reason="CPU features are masked only on x86_64."
-)
-@pytest.mark.skipif(
-    cpuid_utils.get_cpu_vendor() != cpuid_utils.CpuVendor.INTEL,
-    reason="CPU templates are only available on Intel.",
-)
 @pytest.mark.skipif(
     utils.get_kernel_version(level=1) not in SUPPORTED_KERNELS,
     reason=f"Supported kernels are {SUPPORTED_KERNELS}",
 )
-@pytest.mark.parametrize("cpu_template", SNAPSHOT_RESTORE_SHARED_NAMES["cpu_templates"])
 @pytest.mark.timeout(900)
 @pytest.mark.nonci
-def test_cpu_wrmsr_snapshot(bin_cloner_path, cpu_template):
+def test_cpu_wrmsr_snapshot(bin_cloner_path, msr_cpu_template):
```

Comment:
There doesn't seem to be any users of the `cpu_template` fixture ... Does it make sense to have two? or should we consolidate them into one?

Reply:
Not all templates supported by test_feat_parity.py (C3, T2, T2CL, T2S, T2A) are supported by test_cpu_features (T2A, T2CL,T2S) so, we cannot use cpu_template fixture for rdmsr/wrmsr_cpuid snapshot/restore tests.
msr_cpu_template fixture was created local to test_cpu_features.py just for this reason.

Reply:
Sure but is cpu_template used anywhere else now?

Reply:
test_feat_parity.py is the only one that is using it right now